controllers/mysql_conf.go

```diff
@@ -0,0 +1,178 @@
+package controllers
+
+import (
+	"fmt"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/cybozu-go/moco"
+)
+
+var (
+	// Default options of mysqld section
+	defaultMycnf = map[string]string{
+		"tmpdir":                              moco.TmpPath,
+		"innodb_tmpdir":                       moco.TmpPath,
+		"character_set_server":                "utf8mb4",
+		"collation_server":                    "utf8mb4_unicode_ci",
+		"skip_character_set_client_handshake": "ON",
+
+		"default_time_zone": "+0:00",
+
+		"back_log":            "900",
+		"max_connections":     "5000",
+		"max_connect_errors":  "10",
+		"max_allowed_packet":  "1G",
+		"max_heap_table_size": "64M",
+		"sort_buffer_size":    "4M",
+		"join_buffer_size":    "2M",
+		"thread_cache_size":   "100",
+		"wait_timeout":        "604800", // == 7 days
+		"lock_wait_timeout":   "60",
+
+		"table_open_cache":       "65536", // The default setting causes MY-012102
+		"table_definition_cache": "65536", // mitigate a innodb table cache eviction.
+
+		"transaction_isolation": "READ-COMMITTED",
+		"tmp_table_size":        "64M",
+		"slow_query_log":        "ON",
+		"long_query_time":       "2",
+		"log_error_verbosity":   "3",
+
+		"max_sp_recursion_depth": "20",
+
+		"print_identified_with_as_hex": "ON",
+
+		"loose_binlog_transaction_compression": "ON", // It would reduce the size of binlog by a third, but only available in 8.0.20 or later.
+
+		"information_schema_stats_expiry": "0", // No need to cache information_schema.tables values
+
+		// MyISAM Specific options
+		"key_buffer_size":           "32M",
+		"read_buffer_size":          "2M",
+		"read_rnd_buffer_size":      "4M",
+		"bulk_insert_buffer_size":   "8M",
+		"myisam_sort_buffer_size":   "4M",
+		"myisam_max_sort_file_size": "128M",
+		"myisam_repair_threads":     "1",
+		"myisam_recover_options":    "FORCE,BACKUP",
```

Comment:
MyISAM tables are not transactional and therefore is not
suitable for use in MOCO clusters.

Why don't you disable MyISAM altogether by `disabled_storage_engines="MyISAM"`?